theano/gpuarray/dnn_base.c

```diff
@@ -88,6 +92,13 @@ c_set_filter(PyGpuArrayObject *var, cudnnFilterDescriptor_t desc) {
     dims[i] = PyGpuArray_DIM(var, i);
   }
 
+  /* Filters can't be less than 3d so we pad */
```

Comment:
Can you add in this comment and the previous like this which current version of cudnn have this limit. In case it change in the futur, it is useful.